crates/matrix-sdk/src/account.rs

```diff
@@ -128,14 +128,30 @@ impl Account {
     /// ```
     pub async fn get_avatar_url(&self) -> Result<Option<OwnedMxcUri>> {
         let user_id = self.client.user_id().ok_or(Error::AuthenticationRequired)?;
+        let user_id = user_id.to_owned();
```

Comment:
This `to_owned()` isn't necessary anymore if the `save_avatar_url` accepts a `&UserId` instead of `&OwnedUserId`.